packages/react/src/types.ts

```diff
@@ -31,5 +31,65 @@ export type CssObject<TProps> = Readonly<{
 export type CssFunction<TProps = unknown> =
   | CssType<TProps>
   | BasicTemplateInterpolations // CSS values in tagged template expression
+  | null
```

Comment:
I think I'm missing a bit of context here - in what kind of situation would we get `null`? Is it something like `null && styles`?

Reply:
Yes doing conditional styles (ternary / binary / etc).

Reply:
I will note that `null` isn't present in some types, eg. in the `ac()` functions.  Does it need wider propagation?

Reply:
Let me take a look. In this case it might not matter because it's all falsy values 🤔.